Detectors/ITSMFT/MFT/condition/include/MFTCondition/DCSNameResolver.h

```diff
@@ -0,0 +1,234 @@
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel001.actual.iMon", "MFT_RU_LV/H1/D0/F1/Z1/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel002.actual.iMon", "MFT_RU_LV/H1/D0/F1/Z2/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel003.actual.iMon", "MFT_RU_LV/H1/D0/F1/Z3/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel004.actual.iMon", "MFT_RU_LV/H1/D1/F1/Z0/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel005.actual.iMon", "MFT_RU_LV/H1/D1/F1/Z1/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel006.actual.iMon", "MFT_RU_LV/H1/D1/F1/Z2/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel007.actual.iMon", "MFT_RU_LV/H1/D1/F1/Z3/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel008.actual.iMon", "MFT_RU_LV/H1/D2/F1/Z0/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel009.actual.iMon", "MFT_RU_LV/H1/D2/F1/Z1/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel010.actual.iMon", "MFT_RU_LV/H1/D2/F1/Z2/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard10/channel011.actual.iMon", "MFT_RU_LV/H1/D2/F1/Z3/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel000.actual.iMon", "MFT_RU_LV/H1/D3/F1/Z0/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel001.actual.iMon", "MFT_RU_LV/H1/D3/F1/Z1/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel002.actual.iMon", "MFT_RU_LV/H1/D3/F1/Z2/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel003.actual.iMon", "MFT_RU_LV/H1/D3/F1/Z3/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel004.actual.iMon", "MFT_RU_LV/H1/D4/F1/Z0/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel005.actual.iMon", "MFT_RU_LV/H1/D4/F1/Z1/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel006.actual.iMon", "MFT_RU_LV/H1/D4/F1/Z2/iMon");
+    dictFull2Alias.emplace("mft_infra:CAEN/alimftcae001/branchController11/easyCrate1/easyBoard15/channel007.actual.iMon", "MFT_RU_LV/H1/D4/F1/Z3/iMon");
+  }
+
+  std::string getFullName(std::string alias)
```

Comment:
`std::string& getFullName(const std::string& alias)`

Reply:
Is there a specific reason to use the address operator?

Reply:
`const type&`  passes the reference to object type instead of returning a copy, which consumes memory. `const` prevents modification of the returned object.